zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/ErrorMapperTest.java

```diff
@@ -286,6 +287,11 @@ public ProcessInstanceServices processInstanceService() {
       return Mockito.mock(ProcessInstanceServices.class);
     }
 
+    @Bean
+    public GroupController groupController() {
+      return Mockito.mock(GroupController.class);
+    }
+
```

Comment:
❌ Adjusting this class will become obsolete soon after https://github.com/camunda/camunda/pull/19978 is merged 👍 🙂